STATUS: CLEANER DOES NOT FIX
MODIFIED: N/A
MODIFIED FILE: N/A
CLEANER: io.cloudslang.fortest.SensitiveValueTest.testEncryptedStringSensitiveValuePreEncrypted
CLEANER FILE: /Users/chopinli/Documents/UT-Austin/test/automated_script/CloudSlang/cloud-slang/cloudslang-entities/src/test/java/io/cloudslang/fortest/SensitiveValueTest.java
POLLUTER: N/A
POLLUTER FILE: N/A
ORIGINAL CLEANER SIZE: 18
NEW CLEANER SIZE: 18
ELAPSED TIME: 0.0
==========================
@@ -0,0 +0,24 @@
+ {
+     {
+     }
+     {
+     }
+     final String encryptedString = "{Encrypted}bar";
+     final String expectedDecryptedValue = "bar";
+     SensitiveValue value = ValueFactory.createEncryptedString(encryptedString, true);
+     verifyEncrypted(value, expectedDecryptedValue, encryptedString);
+     value.encrypt();
+     verifyEncrypted(value, expectedDecryptedValue, encryptedString);
+     value.decrypt();
+     verifyDecrypted(value, expectedDecryptedValue);
+     value.decrypt();
+     verifyDecrypted(value, expectedDecryptedValue);
+     value.encrypt();
+     verifyEncrypted(value, expectedDecryptedValue, encryptedString);
+     value.encrypt();
+     verifyEncrypted(value, expectedDecryptedValue, encryptedString);
+     {
+     }
+     {
+     }
+ }
